scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -135,8 +138,8 @@ is_ios_running_on_remote_node() {
 # This function will initialize values of local and remote ioservices 
 # fid variable.
 get_ios_fid() {
-    LOCAL_IOS_FID=$(./prov-m0-reset $CDF_FILENAME $HA_ARGS_FILENAME --get-fid ios $LOCAL_NODE $SINGLE_NODE_RUNNING)
-    REMOTE_IOS_FID=$(./prov-m0-reset $CDF_FILENAME $HA_ARGS_FILENAME --get-fid ios $REMOTE_NODE $SINGLE_NODE_RUNNING)
+    LOCAL_IOS_FID=$(cluster_utility_function $CDF_FILENAME $HA_ARGS_FILENAME --get-fid ios $LOCAL_NODE $SINGLE_NODE_RUNNING)
```

Comment:
Create a PR in ha/ to add needed funtionality.

Reply:
This is provided in cluster_utility_function file for now. Mandar said he will have them in some common functions file but that will not be done by LDR-R1

Reply:
Changed as per our discussion.